nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -602,3 +602,57 @@ To delete a Dynamic Group Membership you would send a `DELETE` request to the de
+
+Creating, updating, or deleting a Dynamic Group will automatically invalidate the cache for that group. This means that the next time the group is evaluated, the cache will be refreshed with the new membership information.
+
+This greatly speeds up the reverse association of an object to the Dynamic Group(s) it is a part of.
+
+A Dynamic Group object in the ORM exposes two (2) properties for retrieving the members of that group:
+
+- `members` - The evaluated QuerySet defined by the Dynamic Group and it's potential child groups. This will always perform database queries.
+- `members_cached` - A cached instance of the `members` property. This will only perform database queries if the cache is expired.
+
+Additionally a method is exposed for retrieving the members of a Dynamic Group:
+
+- `get_members()` - A way of retrieving the members of a Dynamic Group and also forcibly refreshing the cache. The arguments are:
+    - `skip_cache` - A boolean value that will ignore the cache and will always return the `members` QuerySet if `True`, defaults to `False`.
+    - `force_update_cache` - A boolean value that will force the cache to be updated with the current membership of the Dynamic Group if `True`, defaults to `False`.
+
+An model instance that supports Dynamic Groups will expose the following properties:
+
+- `dynamic_groups` - Full reverse membership of instance to `DynamicGroup` objects, performs the most database queries.
+- `dynamic_groups_cached` - Full reverse membership of instance to `DynamicGroup` objects, uses cached member list if available. Ideal for most use cases.
+- `dynamic_groups_list` - List of membership of instance to `DynamicGroup` objects, performs one less database query than `dynamic_groups`.
+- `dynamic_groups_list_cached` - List of membership of instance to `DynamicGroup` objects, uses cached member list if available. Performs no database queries in optimal conditions.
+
+The model instance will also expose the following methods:
+
+- `get_dynamic_groups` - A way of retrieving the Dynamic Groups the instance is a member of. The arguments are:
+    - `skip_cache` - A boolean value that will ignore the cache(s) if `True`, defaults to `False`.
+    - `as_queryset` - A boolean value that will return the `dynamic_groups` QuerySet instead of the `dynamic_groups_list` list if `True`, defaults to `False`.
+
+### Invalidating/Refreshing the Cache
+
+If you need to invalidate the membership cache for a Dynamic Group, you can do so by running the management command: `nautobot-server refresh_dynamic_group_member_caches`. This will invalidate the cache for all Dynamic Groups.
```

Comment:
Might be useful to think about adding a "Refresh" button on the Dynamic Group detail view? I know editing and saving the group will have the same effect, but that is more clicks for the user and also results in unnecessarily creating a changelog entry.

Reply:
@lampwins said last week we can add as follow-on